spec/fixtures/site_settings/locale_override.yml

```diff
@@ -0,0 +1,14 @@
+basic:
+  default_locale:
+    default: 'en'
+    enum: 'LocaleSiteSetting'
+
+search:
+  min_search_term_length:
+    client: true
+    default:
+      development: 1
```

Comment:
I am not a fan of this part of the feature. Let's take it out, default per environment is a recipe for confusion and bugs.